llvm/lib/Target/AMDGPU/SIPeepholeSDWA.cpp

```diff
@@ -278,11 +281,16 @@ static void copyRegOperand(MachineOperand &To, const MachineOperand &From) {
   }
 }
 
+static bool isSameReg(const MachineOperand &Op, Register Reg) {
+  return Op.isReg() && Op.getReg() == Reg;
+}
+
+static bool isSameReg(const MachineOperand &Op, Register Reg, unsigned SubReg) {
```

Comment:
Probably should just use TargetInstrInfo::RegSubRegPair

Reply:
This overload of `isSameReg` was just introduced for consistency and is not really necessary.